scripts/release/build_all.sh

```diff
@@ -71,9 +66,13 @@ docker run --rm --gpus all                  \
             -v ./scripts:/scripts           \
             icicle-release-ubi8-cuda122 bash /scripts/release/build_release_and_tar.sh icicle_$version ubi8 cuda122 &
 
+# NOTE: The last command should not be run in the background,
```

Comment:
you are correct here but what happens if this one is finished but the previous ones still did not? maybe need some way to wait for all.

Reply:
Fixed in commit [d8dfba9](https://github.com/ingonyama-zk/icicle/pull/690/commits/d8dfba99740a15eda87ba14a81fa61483018f2ea)